frotz.py

```diff
@@ -126,15 +131,34 @@ def start_game(self, game_file):
             # print a welcome banner and pause for a moment
             # that way the end user will have a chance to see which version of frotz we are using
             # and which file we are loading
-
-            if platform.machine().startswith('arm'):
-                logging.error('ARM not supported, yet') # FIXME
-                sys.exit(0)
+            cmd = '/usr/games/frotz'
```

Comment:
Are you sure `frotz` is always installed in `/usr/games/frotz` across distros?

Reply:
Yeah , I have checked on Debian and Ubuntu, Not sure about fedora. Do you use Fedora, Can you check once?

Reply:
Fedora saves it to the /usr/bin as other distributions do. Only Linux distros that use `apt` saves it to `/usr/games`. All others, Fedora, RHEL, CentOS, Arch Linux and Manjaro installs it to `/usr/bin`

Reply:
Thanks Done in 5c38b6c

Reply:
/usr/games is in $PATH on Debian and Ubuntu, so use $PATH.